tournament/tournamentServerThread.py

```diff
@@ -222,10 +225,11 @@ def sendReply(self, action, *args, **kwargs) :
                     self.socket.write(reply)
 
 
-        except :
+        except:
                 self.log.exception("Something awful happened when sending reply !")  
-  
-    def done(self) :
+
+
+    def done(self):
         self.parent.removeUpdater(self)
         if self.socket:
-            self.socket.deleteLater()
+            self.socket.deleteLater()
```

Comment:
Would you mind setting up your IDE to preserve and insert the newline at EOL? This is done through `Settings->Editor->Other->Ensure line feed at file end on save.`